Build failed: multiple definition of `dummy'

```
file config.log: writing...
file configure.local: no (fully automatic configuration)
tested operating system: Linux -> OSENUM=MANDOC_OS_OTHER
tested cc -W: tested noop: yes
selected CFLAGS="-g -W -Wall -Wmissing-prototypes -Wstrict-prototypes -Wwrite-strings -Wno-unused-parameter"
tested noop-static: yes
selected STATIC="-static"
tested dirent-namlen: no
tested be32toh: yes
tested be32toh-DSYS_ENDIAN: no
tested EFTYPE: no
tested err: yes
tested getline: yes
tested getsubopt: yes
tested isblank: yes
tested mkdtemp: yes
tested ntohl: yes
tested O_DIRECTORY: yes
tested PATH_MAX: yes
tested pledge: no
tested sandbox_init: no
tested progname: no
tested reallocarray: yes
tested recallocarray-D_OPENBSD_SOURCE: no
tested rewb-bsd: no
tested rewb-sysv: yes
tested strcasestr-D_GNU_SOURCE: yes
tested stringlist: no
tested strlcat: no
tested strlcpy: no
tested strndup: yes
tested strptime-D_GNU_SOURCE: yes
tested strsep: yes
tested strtonum-D_OPENBSD_SOURCE: no
tested vasprintf-D_GNU_SOURCE: yes
tested fts-DFTS_COMPARE_CONST: no
tested fts: yes
tested less -T: yes
selected UTF8_LOCALE=en_US.utf8
tested wchar-D_GNU_SOURCE-DUTF8_LOCALE="en_US.utf8": yes
tested nanosleep: yes
tested ohash-lutil: no
selected LDADD="    -lz"
file config.h: written
file Makefile.local: written
make: Entering directory '/home/yochanan/Documents/pkgbuilds/mangl-git/src/mangl'
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/compat_err.o mandoc/compat_err.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/compat_fts.o mandoc/compat_fts.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/compat_getline.o mandoc/compat_getline.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/compat_getsubopt.o mandoc/compat_getsubopt.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/compat_isblank.o mandoc/compat_isblank.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/compat_mkdtemp.o mandoc/compat_mkdtemp.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/compat_ohash.o mandoc/compat_ohash.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/compat_progname.o mandoc/compat_progname.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/compat_reallocarray.o mandoc/compat_reallocarray.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/compat_recallocarray.o mandoc/compat_recallocarray.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/compat_strcasestr.o mandoc/compat_strcasestr.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/compat_strlcat.o mandoc/compat_strlcat.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/compat_strlcpy.o mandoc/compat_strlcpy.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/compat_strndup.o mandoc/compat_strndup.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/compat_strsep.o mandoc/compat_strsep.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/compat_strtonum.o mandoc/compat_strtonum.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/compat_vasprintf.o mandoc/compat_vasprintf.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/man.o mandoc/man.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/man_macro.o mandoc/man_macro.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/man_validate.o mandoc/man_validate.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/att.o mandoc/att.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/lib.o mandoc/lib.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/mdoc.o mandoc/mdoc.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/mdoc_argv.o mandoc/mdoc_argv.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/mdoc_macro.o mandoc/mdoc_macro.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/mdoc_state.o mandoc/mdoc_state.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/mdoc_validate.o mandoc/mdoc_validate.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/st.o mandoc/st.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/eqn.o mandoc/eqn.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/roff.o mandoc/roff.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/roff_validate.o mandoc/roff_validate.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/tbl.o mandoc/tbl.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/tbl_data.o mandoc/tbl_data.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/tbl_layout.o mandoc/tbl_layout.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/tbl_opts.o mandoc/tbl_opts.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/arch.o mandoc/arch.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/chars.o mandoc/chars.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/mandoc.o mandoc/mandoc.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/mandoc_aux.o mandoc/mandoc_aux.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/mandoc_msg.o mandoc/mandoc_msg.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/mandoc_ohash.o mandoc/mandoc_ohash.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/mandoc_xr.o mandoc/mandoc_xr.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/msec.o mandoc/msec.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/preconv.o mandoc/preconv.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -D_FORTIFY_SOURCE=2  -c -o mandoc/read.o mandoc/read.c
cc -g -O2 -Wall -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -o mangl mandoc/compat_err.o mandoc/compat_fts.o mandoc/compat_getline.o mandoc/compat_getsubopt.o mandoc/compat_isblank.o mandoc/compat_mkdtemp.o mandoc/compat_ohash.o mandoc/compat_progname.o mandoc/compat_reallocarray.o mandoc/compat_recallocarray.o mandoc/compat_strcasestr.o mandoc/compat_strlcat.o mandoc/compat_strlcpy.o mandoc/compat_strndup.o mandoc/compat_strsep.o mandoc/compat_strtonum.o mandoc/compat_vasprintf.o mandoc/man.o mandoc/man_macro.o mandoc/man_validate.o mandoc/att.o mandoc/lib.o mandoc/mdoc.o mandoc/mdoc_argv.o mandoc/mdoc_macro.o mandoc/mdoc_state.o mandoc/mdoc_validate.o mandoc/st.o mandoc/eqn.o mandoc/roff.o mandoc/roff_validate.o mandoc/tbl.o mandoc/tbl_data.o mandoc/tbl_layout.o mandoc/tbl_opts.o mandoc/arch.o mandoc/chars.o mandoc/mandoc.o mandoc/mandoc_aux.o mandoc/mandoc_msg.o mandoc/mandoc_ohash.o mandoc/mandoc_xr.o mandoc/msec.o mandoc/preconv.o mandoc/read.o mandoc/tree.c mandoc/mdoc_term.c mandoc/man_term.c mandoc/tbl_term.c mandoc/tag.c mandoc/roff_term.c mandoc/eqn_term.c mandoc/term_ascii.c mandoc/term.c mandoc/term_tab.c mandoc/out.c hashmap.c main.c -lm -lglut -lz -lGL -lfreetype 
/usr/sbin/ld: mandoc/compat_fts.o:/home/yochanan/Documents/pkgbuilds/mangl-git/src/mangl/mandoc/compat_fts.c:5: multiple definition of `dummy'; mandoc/compat_err.o:/home/yochanan/Documents/pkgbuilds/mangl-git/src/mangl/mandoc/compat_err.c:5: first defined here
/usr/sbin/ld: mandoc/compat_getline.o:/home/yochanan/Documents/pkgbuilds/mangl-git/src/mangl/mandoc/compat_getline.c:5: multiple definition of `dummy'; mandoc/compat_err.o:/home/yochanan/Documents/pkgbuilds/mangl-git/src/mangl/mandoc/compat_err.c:5: first defined here
/usr/sbin/ld: mandoc/compat_getsubopt.o:/home/yochanan/Documents/pkgbuilds/mangl-git/src/mangl/mandoc/compat_getsubopt.c:5: multiple definition of `dummy'; mandoc/compat_err.o:/home/yochanan/Documents/pkgbuilds/mangl-git/src/mangl/mandoc/compat_err.c:5: first defined here
/usr/sbin/ld: mandoc/compat_isblank.o:/home/yochanan/Documents/pkgbuilds/mangl-git/src/mangl/mandoc/compat_isblank.c:5: multiple definition of `dummy'; mandoc/compat_err.o:/home/yochanan/Documents/pkgbuilds/mangl-git/src/mangl/mandoc/compat_err.c:5: first defined here
/usr/sbin/ld: mandoc/compat_mkdtemp.o:/home/yochanan/Documents/pkgbuilds/mangl-git/src/mangl/mandoc/compat_mkdtemp.c:5: multiple definition of `dummy'; mandoc/compat_err.o:/home/yochanan/Documents/pkgbuilds/mangl-git/src/mangl/mandoc/compat_err.c:5: first defined here
/usr/sbin/ld: mandoc/compat_reallocarray.o:/home/yochanan/Documents/pkgbuilds/mangl-git/src/mangl/mandoc/compat_reallocarray.c:5: multiple definition of `dummy'; mandoc/compat_err.o:/home/yochanan/Documents/pkgbuilds/mangl-git/src/mangl/mandoc/compat_err.c:5: first defined here
/usr/sbin/ld: mandoc/compat_strcasestr.o:/home/yochanan/Documents/pkgbuilds/mangl-git/src/mangl/mandoc/compat_strcasestr.c:5: multiple definition of `dummy'; mandoc/compat_err.o:/home/yochanan/Documents/pkgbuilds/mangl-git/src/mangl/mandoc/compat_err.c:5: first defined here
/usr/sbin/ld: mandoc/compat_strndup.o:/home/yochanan/Documents/pkgbuilds/mangl-git/src/mangl/mandoc/compat_strndup.c:5: multiple definition of `dummy'; mandoc/compat_err.o:/home/yochanan/Documents/pkgbuilds/mangl-git/src/mangl/mandoc/compat_err.c:5: first defined here
/usr/sbin/ld: mandoc/compat_strsep.o:/home/yochanan/Documents/pkgbuilds/mangl-git/src/mangl/mandoc/compat_strsep.c:5: multiple definition of `dummy'; mandoc/compat_err.o:/home/yochanan/Documents/pkgbuilds/mangl-git/src/mangl/mandoc/compat_err.c:5: first defined here
/usr/sbin/ld: mandoc/compat_vasprintf.o:/home/yochanan/Documents/pkgbuilds/mangl-git/src/mangl/mandoc/compat_vasprintf.c:5: multiple definition of `dummy'; mandoc/compat_err.o:/home/yochanan/Documents/pkgbuilds/mangl-git/src/mangl/mandoc/compat_err.c:5: first defined here
collect2: error: ld returned 1 exit status
make: *** [Makefile:72: mangl] Error 1
make: Leaving directory '/home/yochanan/Documents/pkgbuilds/mangl-git/src/mangl'
```

--- a/mandoc/compat_err.c
+++ b/mandoc/compat_err.c
@@ -2,7 +2,6 @@
 
 #if HAVE_ERR
 
-int dummy;
 
 #else
 

--- a/mandoc/compat_fts.c
+++ b/mandoc/compat_fts.c
@@ -2,7 +2,6 @@
 
 #if HAVE_FTS
 
-int dummy;
 
 #else
 

--- a/mandoc/compat_getline.c
+++ b/mandoc/compat_getline.c
@@ -2,7 +2,6 @@
 
 #if HAVE_GETLINE
 
-int dummy;
 
 #else
 

--- a/mandoc/compat_getsubopt.c
+++ b/mandoc/compat_getsubopt.c
@@ -2,7 +2,6 @@
 
 #if HAVE_GETSUBOPT
 
-int dummy;
 
 #else
 

--- a/mandoc/compat_isblank.c
+++ b/mandoc/compat_isblank.c
@@ -2,7 +2,6 @@
 
 #if HAVE_ISBLANK
 
-int dummy;
 
 #else
 

--- a/mandoc/compat_mkdtemp.c
+++ b/mandoc/compat_mkdtemp.c
@@ -2,7 +2,6 @@
 
 #if HAVE_MKDTEMP
 
-int dummy;
 
 #else
 

--- a/mandoc/compat_ohash.c
+++ b/mandoc/compat_ohash.c
@@ -2,7 +2,6 @@
 
 #if HAVE_OHASH
 
-int dummy;
 
 #else
 

--- a/mandoc/compat_progname.c
+++ b/mandoc/compat_progname.c
@@ -2,7 +2,6 @@
 
 #if HAVE_PROGNAME
 
-int dummy;
 
 #else
 

--- a/mandoc/compat_reallocarray.c
+++ b/mandoc/compat_reallocarray.c
@@ -2,7 +2,6 @@
 
 #if HAVE_REALLOCARRAY
 
-int dummy;
 
 #else
 

--- a/mandoc/compat_recallocarray.c
+++ b/mandoc/compat_recallocarray.c
@@ -2,7 +2,6 @@
 
 #if HAVE_RECALLOCARRAY
 
-int dummy;
 
 #else
 

--- a/mandoc/compat_strcasestr.c
+++ b/mandoc/compat_strcasestr.c
@@ -2,7 +2,6 @@
 
 #if HAVE_STRCASESTR
 
-int dummy;
 
 #else
 

--- a/mandoc/compat_stringlist.c
+++ b/mandoc/compat_stringlist.c
@@ -2,7 +2,6 @@
 
 #if HAVE_STRINGLIST
 
-int dummy;
 
 #else
 

--- a/mandoc/compat_strlcat.c
+++ b/mandoc/compat_strlcat.c
@@ -2,7 +2,6 @@
 
 #if HAVE_STRLCAT
 
-int dummy;
 
 #else
 

--- a/mandoc/compat_strlcpy.c
+++ b/mandoc/compat_strlcpy.c
@@ -2,7 +2,6 @@
 
 #if HAVE_STRLCPY
 
-int dummy;
 
 #else
 

--- a/mandoc/compat_strndup.c
+++ b/mandoc/compat_strndup.c
@@ -2,7 +2,6 @@
 
 #if HAVE_STRNDUP
 
-int dummy;
 
 #else
 

--- a/mandoc/compat_strsep.c
+++ b/mandoc/compat_strsep.c
@@ -2,7 +2,6 @@
 
 #if HAVE_STRSEP
 
-int dummy;
 
 #else
 

--- a/mandoc/compat_strtonum.c
+++ b/mandoc/compat_strtonum.c
@@ -2,7 +2,6 @@
 
 #if HAVE_STRTONUM
 
-int dummy;
 
 #else
 

--- a/mandoc/compat_vasprintf.c
+++ b/mandoc/compat_vasprintf.c
@@ -2,7 +2,6 @@
 
 #if HAVE_VASPRINTF
 
-int dummy;
 
 #else
 

